k8s.io/test.py

```diff
@@ -344,6 +344,7 @@ def assert_body_url(self, url, expected_content_url):
         self.assertMultiLineEqual(body, expected_body)
 
     def test_go_get(self):
+        # FIXME: need to fix this if we keep this idea.
```

Comment:
does this need to be fixed?

Reply:
i can help with this

Reply:
@thockin here's a fix for the test.py - https://github.com/dims/k8s.io/commit/e09f0cb1f71dd8dfbcc10e97f71dc128387480b1#diff-58c10258360cbb1470635b52e0887e93